File: answers/DhruvPandey/Day6/que1.cpp
#include<iostream>
#include<conio.h>
using namespace std;
int main()
{
int i,j,k;
int target[5];
int num[5]={0,1,2,3,4};
int index[5]={0,1,2,2,1};

for(i=0;i<5;i++)
{
target[index[i]]=num[i];
}
cout<<"NUMS ARE :"<<"\t";
for(k=0;k<5;k++)
{
cout<<num[k]<<"  ";
}
cout<<"\n";
cout<<"INDEX IS :"<<"\t";
for(k=0;k<5;k++)
{
cout<<index[k]<<"  ";
}
cout<<"\n";
cout<<"TARGET IS :"<<"\t";
for(k=0;k<5;k++)
{
cout<<target[k]<<"  ";
}
return 0;
}

File: answers/Aryan Goyal/Day 9/Que 1.py
class Solution:
    def twoSum(self, nums: List[int], target: int):
        l=[]
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                if nums[i]+nums[j]==target:
                    l=[i,j]
        return l
